dotCMS/src/main/java/com/dotmarketing/servlets/ShortyServlet.java

```diff
@@ -102,14 +101,10 @@ private void _serve(final HttpServletRequest request, final HttpServletResponse
     boolean jpegp = jpeg && uri.contains("jpegp");
     boolean isImage = jpeg || w+h>0;
     
-    
-    
-    
-
 
     Optional<ShortyId> shortOpt = APILocator.getShortyAPI().getShorty(id);
     User user = WebAPILocator.getUserWebAPI().getLoggedInFrontendUser(request);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "user". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "user" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)